tests/integration/targets/sysrc/tasks/main.yml

```
@@ -369,14 +369,79 @@
               - "value_2 == sysrc_equals_sign_2.value"
               - "value_2 == conf.spamd_flags"
 
+    ##
+    ## sysrc - #10004 state=absent when using default settings will report `changed=true`
+    ##
+    - name: Test that a key from /etc/defaults/rc.conf is not used to mark changed
+      sysrc:
+        name: dumpdev
+        state: absent
+        path: /tmp/10004.conf
+      register: sysrc_10004_absent
+      failed_when: sysrc_10004_absent.changed
+
+    - name: Test that a delimited key from /etc/defaults/rc.conf is not used to mark changed
+      sysrc:
+        name: rc_conf_files
+        state: value_absent
+        path: /tmp/10004.conf
+      register: sysrc_10004_value_absent
+      failed_when: sysrc_10004_value_absent.changed
+
+    - name: Test that a key from /etc/defaults/rc.conf is not used to mark changed without a path
+      sysrc:
+        name: static_routes
+        state: absent
+      register: sysrc_absent_default
+      failed_when: sysrc_absent_default.changed
+
+    ##
+    ## sysrc - #10394 Ensure that files with multi-line values work
+    ##
+    - name: Copy 10394.conf
+      copy:
+        src: 10394.conf
+        dest: /tmp/10394.conf
+
+    - name: Change value for k1
+      sysrc:
+        name: k1
+        value: v2
+        path: /tmp/10394.conf
+      register: sysrc_10394_changed
+
+    - name: Get file content
+      shell: "cat /tmp/10394.conf"
+      register: sysrc_10394_content
+
+    - name: Ensure sysrc changed k1 from v1 to v2
+      assert:
+        that:
+          - sysrc_10394_changed.changed
+          - >
+            'k1="v2"' in sysrc_10394_content.stdout_lines
+
+    ##
+    ## sysrc - additional tests
+    ##
+    - name: Ensure failure on OID style name since sysrc does not support them
+      sysrc:
+        name: not.valid.var
+        value: test
+      register: sysrc_name_check
+      failed_when:
+        - sysrc_name_check is not failed
+        - >
+          'Name may only contain alpha-numeric and underscore characters' != sysrc_name_check.msg
```

Nit pick.

We do not gain much by breaking this into the other line.
```suggestion
        - 'Name may only contain alpha-numeric and underscore characters' != sysrc_name_check.msg
```

Feel free to disregard it for this PR, though.

---

This line would be invalid YAML, the `>` is needed to make YAML not interpret the `'`.